modules/indexer/code/bleve/bleve.go

```diff
@@ -191,7 +191,7 @@ func (b *Indexer) addUpdate(ctx context.Context, batchWriter git.WriteCloserErro
 		return err
 	} else if !typesniffer.DetectContentType(fileContents).IsText() {
 		// FIXME: UTF-16 files will probably fail here
-		return nil
+		fileContents = make([]byte, 0)
```

Comment:
```suggestion
		// Even if the file is not recognized as a "text file", we could still put its name into the indexers to make the filename become searchable, while leave the content to empty.
		fileContents = nil
```

Does `fileContents = nil` work? In most cases, Golang should treat nil as empty byte slice equally.